.github/workflows/ci.yml

```diff
@@ -43,8 +43,28 @@ jobs:
       - name: Test package
         run: docker run --rm ${{ env.TEST_IMAGE }} "poe test"
 
-      # TODO setup coverage in #8
-#      - name: Run codacy coverage reporter
-#        uses: codacy/codacy-coverage-reporter-action@v1
-#        with:
-#          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+  docs-test:
+    name: Test documentation
+    runs-on: ubuntu-latest
```

Comment:
So steps are github actions, right?

Reply:
Steps are like jenkins stages and jobs are like jenkins pipelines and workflow is a collection of jobs based on a specific trigger.  

One can run a Github action in a step but i also think that they call the whole CI system "GitHub Actions"